src/clr-angular/data/datagrid/datagrid-column-separator.ts

```diff
@@ -1,43 +1,86 @@
 /*
- * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import { DOCUMENT } from '@angular/common';
-import { Component, Inject, Renderer2 } from '@angular/core';
+import { AfterViewInit, Component, ElementRef, Inject, OnDestroy, Renderer2, ViewChild } from '@angular/core';
 
 import { ClrDragEvent } from '../../utils/drag-and-drop/drag-event';
 import { ColumnResizerService } from './providers/column-resizer.service';
 import { TableSizeService } from './providers/table-size.service';
-import { UNIQUE_ID_PROVIDER, UNIQUE_ID } from '../../utils/id-generator/id-generator.service';
+import { UNIQUE_ID, UNIQUE_ID_PROVIDER } from '../../utils/id-generator/id-generator.service';
+import { DragEventType } from '../../utils/drag-and-drop/interfaces/drag-event.interface';
+import { DomAdapter } from '../../utils/dom-adapter/dom-adapter';
+import { ClrCommonStringsService } from '../../utils/i18n/common-strings.service';
+import { commonStringsDefault } from '@clr/core';
 
 @Component({
   selector: 'clr-dg-column-separator',
   template: `
-    <div class="datagrid-column-handle" aria-hidden="true"
-      clrDraggable 
-      [clrGroup]="columnSeparatorId" 
-      (clrDragStart)="showTracker(resizeTrackerEl)" 
-      (clrDragMove)="moveTracker($event, resizeTrackerEl)" 
-      (clrDragEnd)="hideTracker(resizeTrackerEl)"></div>
+    <div class="datagrid-column-handle" aria-live="polite" tabindex="0" #columnHandle
+      clrDraggable
+      [clrGroup]="columnSeparatorId"
+      (clrDragStart)="showTracker(resizeTrackerEl)"
+      (clrDragMove)="moveTracker($event, resizeTrackerEl)"
+      (clrDragEnd)="hideTracker(resizeTrackerEl)"
+      (keydown)="onKeyPress($event)"
```

Comment:
Unfortunately, we cannot use these template even bindings on `keydown` and `keyup` events here as they would trigger change detection on each `keydown` event. We need to use `Renderer2.listen` with `NgZone.runOutisdeAngular`